webpack/webpack.ghpages.cjs

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { merge } = require('webpack-merge');
+
+const generateConfig = require('./webpack.config.cjs');
+const constants = require('./constants.cjs');
+
+// Add timerbar_test.html
+constants.cactbotModules['timerbarTest'] = 'ui/test/timerbar_test';
+constants.cactbotHtmlChunksMap['ui/test/timerbar_test.html'] = {
+  chunks: [
+    constants.cactbotModules.timerbarTest,
+  ],
+};
+
+const baseConfig = generateConfig(constants);
```

Comment:
Hmm.  Is it possible to have ghpages be based on prod? Mostly I would like prod and ghpages to be pretty much identical, maybe minus the source map, and want changes to one to be reflected in the other.

Reply:
Yes, I'll tweak it like that.